packages/pds/src/services/auth.ts

```diff
@@ -91,7 +91,6 @@ export class AuthService {
     this.db.assertTransaction()
     const token = await this.db.db
       .selectFrom('refresh_token')
-      .if(this.db.dialect !== 'sqlite', (qb) => qb.forUpdate())
```

Comment:
To mitigate this, we might shorten the distance between this select and the update below.  Or perform the update with an additional match on the selected `expiredAt` time to wrangle concurrent refreshes.

Reply:
this gets tweaked in the account manager code. so i'll give it a look over there to make sure it all makes sense 👌

Reply:
okay got this into the account manager PR

you can check it out here: https://github.com/bluesky-social/atproto/pull/1769/files#r1378166057